### ISSUE TITLE
Embossed text and curved text

### ISSUE THREAD

**Issue by AmarendarAndhe** [Bartzi/see#93]
@Bartzi , I did cropped text region and I am using these cropped images for text extraction. But I am unable to extract text using your text_recoginition_demo.py, i have images like black-on-black text which are embossed text and curved text and font is different. Could you please suggest an idea how to achieve success on these type of images for text extraction. Thank you

**Comment by Bartzi** [comment]
hmm, I don't know how different the appearance of your samples is from the appearance of the samples we used for training, but this might bee the actual problem.

**Comment by AmarendarAndhe** [comment]
hi Bartzi,

Hope you are doing good!

Sorry for late response.

Exact samples i can not share because of compliance issues but i can say these samples look like tyre images.

I trained pre-trained model with these images, but results are not accurate because of embossed text (text and background have same color)

I tried almost all image pre-processing techniques but output is not clearly differentiating text and the background.

Could you please suggest some ideas on how can i achieve good accuracy?

Thank you,
Amar

**Comment by Bartzi** [comment]
Hi,

sry for my late reply, either^^
Do you have an example that you can show me, which is close to the ones you are experimenting with? It is very difficult for me to imagine what you are exactly trying to do and how one might approach this problem with our proposed model. 

Do you have a full tyre and your job is to localize the text? Or do you have already cropped word images and now you only want to recognize the text in these cropped images?

**Comment by AmarendarAndhe** [comment]
hi Bartzi,

I already cropped word images and now i only want to recognize the text in these cropped images. Below is the sample cropped word images.
![CW_3180_0](https://user-images.githubusercontent.com/26601838/118858533-e4802100-b8f6-11eb-8b11-86f3eb7d700e.png)
![CW_3202_0](https://user-images.githubusercontent.com/26601838/118858642-04174980-b8f7-11eb-9917-61e1f9b425e1.png)

Please let me know, if you need more details.

Thank you,
Amar

**Comment by Bartzi** [comment]
thank you for sharing some examples!

I think I might have a solution for you.
We published another paper that uses a similar idea as presented in SEE but only for text recognition. You can find the code/models(and also a link to the paper) [here](https://github.com/Bartzi/kiss). This might rather be an approach you are looking for.
However, there is one crucial question: How many images do you have for training?
